### PR TITLE
Feature/stackdriver api migration

### PR DESCRIPTION
**Pull request by anuradha-bajpai-google**
## Pull Request Template
Please, go through these steps before you submit a PR.

1. Make sure that your PR is not a duplicate.
2. **Before** submitting the PR, review your changes:
    * Adjust the code to the existing [style](https://google.github.io/styleguide).
    * Add a set of unit tests for the changes and ensure they all pass.
    * Add accompanying README.md file with instructions on usage. See [awesome-readme](https://github.com/matiassingers/awesome-readme) for good examples of high-quality READMEs.
    * Add a link to your contribution in the top-level [README](https://github.com/GoogleCloudPlatform/professional-services/blob/master/README.md) (alpha-order).
    * Add [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0) license headers with an up-to-date copyright date attributed to `Google LLC`.
    * Remove unnecessary LICENSE files. There's no need to include an additional license since all repository submissions are covered by the top-level Apache 2.0 [license](https://github.com/GoogleCloudPlatform/professional-services/blob/master/LICENSE).
    * **For new tools/examples**, file your project with the PSO Engineering Council.

3. **After** these steps, you're ready to open a pull request.
    * Give a descriptive title to your PR.
    * Provide a description of your changes.
    * Include the Engineering Council bug ID with your pull request.
    * Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
<br/>

**PLEASE REMOVE THIS TEMPLATE BEFORE SUBMITTING**

### CODE REVIEW

File: tools/quota-monitoring-alerting/java/README.md [hunk 1]
```diff
@@ -18,13 +18,13 @@ Quota Monitoring Solution is a stand-alone application of an easy-to-deploy Data
 
 The architecture is built using Google Cloud managed services - Cloud Functions, Pub/Sub, Dataflow and BigQuery. 
 - The solution is architected to scale using Pub/Sub.
-- Cloud Scheduler is used to trigger Cloud Functions. This is also an user interface to configure frequency, parent nodes, alert threshold and email Ids. Parent node could be an organization Id, folder id, list of organization Ids or list of folder Ids.
+- Cloud Scheduler is used to trigger Cloud Functions. This is also an user interface to configure frequency, parent nodes, alert threshold and email Ids. Parent node could be an orgnization Id, folder id, list of organization Ids or list of folder Ids.
```
Comment (iht) [inline tools/quota-monitoring-alerting/java/README.md:21]: Typo in `organization`

File: tools/quota-monitoring-alerting/java/README.md [hunk 2]
```diff
@@ -18,13 +18,13 @@ Quota Monitoring Solution is a stand-alone application of an easy-to-deploy Data
...
 - Cloud Functions are used to scan quotas across projects for the configured parent node.
 - DataFlow is used to load data in BigQuery.
 - BigQuery is used to store data. 
 - Alert threshold will be applicable across all metrics. 
 - Alerts can be received by Email, Mobile App, PagerDuty, SMS, Slack, Web Hooks and Pub/Sub. Cloud Monitoring custom log metric has been leveraged to create Alerts.
-- Easy to get started and deploy with Data Studio Dashboard. In addition to Data Studio, other visualization tools can be configured. 
+- Easy to get started and deploy with Data Studio Dashboard. In addition to Data Studion, other visualization tools can be configured.
```
Comment (iht) [inline tools/quota-monitoring-alerting/java/README.md:27]: Typo in `Studio`

File: tools/quota-monitoring-alerting/java/README.md [hunk 3]
```diff
@@ -310,7 +266,7 @@ gsutil cp gs://quota-monitoring-solution-demo-bucket/terraform.tfvars .
    - terraform/terraform.tfvars
 ### 3.7 Configure Terraform
 1. Open terraform.tfvars file in your favourite editor and change values for the variable 
-2. Values for variable source_code_bucket_name, source_code_zip and source_code_notification_zip are for source code zip in the storage bucket. These are links to the Cloud Function source code. If you want to upgrade to latest code changes everytime you run 'terraform apply', change to this code source repository. DO NOT CHANGE if you do not want to receive latest code changes while running 'terraform apply' everytime after deployment. 
+2. Values for variable source_code_bucket_name, source_code_zip and source_code_notification_zip are for source code zip in the storage bucket. These are links to the Cloud Function source code. If you want to upgrade to latest code changes everytime you run 'terraform apply', change to this code source repository. DO NOT CHANGE if you do not want to recieve latest code changes while running 'terraform apply' everytime after deployment.
```
Comment (iht) [inline tools/quota-monitoring-alerting/java/README.md:269]: Typo in `receive`

**Review by iht — APPROVED** [review]
Please fix the typos in the `README.md` file.

Other than that, LGTM.

**Comment by anuradha-bajpai-google** [comment]
Hi Israel, Thank you for reviewing! Please merge the code.

**Review by iht — APPROVED** [review]

**Pull request merged**